src/main/java/Talky/Task.java

```diff
@@ -0,0 +1,119 @@
+package Talky;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Class that acts as Task
+ */
+public class Task {
+    protected String name;
```

Comment:
The variable marked, as a boolean, should be isMarked to follow the Java Coding Standard